lib/libvfio-user.c

```diff
@@ -1162,34 +1164,20 @@ vfu_realize_ctx(vfu_ctx_t *vfu_ctx)
 }
 
 int
-vfu_ctx_drive(vfu_ctx_t *vfu_ctx)
+vfu_run_ctx(vfu_ctx_t *vfu_ctx)
 {
     int err;
+    bool blocking = !(vfu_ctx->flags & LIBVFIO_USER_FLAG_ATTACH_NB);
 
-    if (vfu_ctx == NULL || !vfu_ctx->realized) {
+    assert(vfu_ctx != NULL);
```

Comment:
the assert must be before the assignment

Reply:
Thanks. I overlooked it.